pkgpanda/util.py

```diff
@@ -29,6 +31,16 @@
 }
 
 
+def rm_r(path):
+    """ Remove the file or directory at <path>. <path> must be a single string, not a collection type. Same semantics as check_call(["rm","-rf",path])"""
```

Comment:
This line is too long and failing the `tox` run. Please write docstrings according to Python conventions, documented here: https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings (a short one-line description with any other necessary details included in following paragraphs).